CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 1.128.0 (July 21, 2023)
+ENHANCEMENTS:
+* resource/spotinst_stateful_node_aws: Added `delete` block in `stateful_node_aws` for aws stateful node deletion
```

Comment:
change to - Modified `deallocation_config` to provide an option to the user to choose whether to keep the underlying resources alive or not